index.html

```diff
@@ -0,0 +1,459 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars" id="menu-button">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav" class="desk-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="menu hide" id="menu-id">
+      <button id="close">
+        <i>X</i>
+      </button>
+      <nav>
+        <ul>
+          <li class="list-mobile"><a href="#portfolio" class="menu-item">Portfolio</a></li>
+          <li class="list-mobile"><a href="#" class="menu-item">About</a></li>
```

Comment:
[OPTIONAL] We understand you have spent much time working on this "Mobile menu" project 😄 . however, you should make sure that when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed. because it will increase the user experience and also it's part of the requirement. 👍
> . When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.